### DISCUSSION TITLE
Welcome to web-dev Discussions!

### DISCUSSION THREAD

**Discussion by liomessi69** [liomessi69/web-dev#8] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TusharPatwal** [comment, marked as answer]
yes

**Comment by YahefuYusufu** [comment]
Hi .
i am looking for a  open project if possible .

**Comment by YahefuYusufu** [comment]
Hi, i am a front end developer with js ,react,typescript,tailwind
